### PR TITLE
chore: migrate to projen

### PR DESCRIPTION
**Pull request by eladb**
*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,29 +1,91 @@
+# Generated by projen. To modify, edit .projenrc.js and run "npx projen".
 name: Release
 on:
   push:
     branches:
       - master
-      - 2.x
```
Comment (iliapolo) [inline .github/workflows/release.yml:6]: This is worth keeping...

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (iliapolo) [inline LICENSE:189]: What happened here?

File: .github/workflows/release.yml (hunk 1 above)
Comment (eladb) [reply .github/workflows/release.yml:6]: Good catch. I'll add to projen.

File: LICENSE (hunk 2 above)
Comment (eladb) [reply LICENSE:189]: Good catch, I'll add to projen.

File: .github/workflows/release.yml (hunk 1 above)
Comment (eladb) [reply .github/workflows/release.yml:6]: Fixed

File: LICENSE (hunk 2 above)
Comment (eladb) [reply LICENSE]: Fixed

**Pull request merged**